server/models/EmailTemplateModel.py

```diff
@@ -1,14 +1,27 @@
+from sqlalchemy_serializer import SerializerMixin
 from app import db 
 
-class EmailTemplateModel(db.Model):
+class EmailTemplateModel(db.Model, SerializerMixin):
```

Comment:
I think templates will need an owner field, and we might want all templates owned by a user to have unique names.